src/events/index.ts

```diff
@@ -13,6 +13,8 @@ const serviceEventsSalt = parseInt(process.env.SERVICE_EVENTS_SALT || '12345');
 const servicePushNotifications = parseInt(
   process.env.SERVICE_PUSH_NOTIFICATIONS || '0'
 );
+const webHookURL =
```

Comment:
Its webhook not web hook
```suggestion
const webhookURL =
```